server/innernet-server@.service

```diff
@@ -2,12 +2,18 @@
 Description=innernet server for %I
 After=network-online.target nss-lookup.target
 Wants=network-online.target nss-lookup.target
+# Disable systemd's unit start rate limiting logic, which could override Restart=always.
+# See https://unix.stackexchange.com/q/289629/352972
+StartLimitIntervalSec=0
 
 [Service]
 Type=simple
 Environment="RUST_LOG=info"
 ExecStart=/usr/bin/innernet-server serve %i
 Restart=always
+# When the daemon exits, wait this amount of secs before restarting. Prevents innernet from
+# start-looping each 100ms for example when there is a problem reaching the server.
+RestartSec=60
```

Comment:
For client, the restart interval of 60 secs is somewhat natural as it is equal to the fetch interval.

But I don't have an idea what the server restart interval should be. Perhaps a bit less?

Reply:
Yeah, I think it should be closer to 1s than 60s.